src/System.Windows.Forms.Primitives/src/System/Windows/Forms/Internals/ThemingScope.cs

```diff
@@ -12,18 +14,19 @@ namespace System.Windows.Forms
     internal static class ThemingScope
     {
         private static Kernel32.ACTCTXW s_enableThemingActivationContext;
-        private static IntPtr s_hActCtx;
+        private static nint s_hActCtx;
         private static bool s_contextCreationSucceeded;
 
         /// <summary>
         ///  We now use explicitactivate everywhere and use this method to determine if we
         ///  really need to activate the activationcontext.  This should be pretty fast.
         /// </summary>
-        private static bool IsContextActive()
+        private unsafe static bool IsContextActive()
         {
+            Foundation.HANDLE current;
             return s_contextCreationSucceeded
-                && Kernel32.GetCurrentActCtx(out IntPtr current).IsTrue()
-                && current == s_hActCtx;
+                && PInvoke.GetCurrentActCtx(&current) == true
```

Comment:
Does `PInvoke.GetCurrentActCtx(...)` return a bool?

Reply:
The conversion is implicit
```suggestion
                && PInvoke.GetCurrentActCtx(&current)
```